modules/auxiliary/scanner/http/docker_enum.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'Docker Server Version Scanner',
+      'Description' => 'This module attempts identify the version of the Docker Server running.',
+      'Author'     => [ 'Agora-Security' ],
+      'License'   => MSF_LICENSE
+    )
+    register_options(
+      [
+        Opt::RPORT(2375),
+      ])
+    register_autofilter_ports([ 2375 ])
```

Comment:
I think you can drop these two lines.

Reply:
Addressed in ffc2f044cc

By the way, I'm doing a commit per comment. Generally I code this way (easier for code review), anyhow I'm not sure if this is ideal. Should I try to group them?